galaxyxml/tool/__init__.py

```diff
@@ -134,49 +131,52 @@ def export(self, keep_old_command=False):
             stdio_element = export_xml.stdios
         except Exception:
             stdio_element = None
-        if not stdio_element:
-            stdio_element = etree.SubElement(export_xml.root, "stdio")
-            etree.SubElement(stdio_element, "exit_code", range="1:", level="fatal")
-        try:
-            export_xml.append(stdio_element)
-        except Exception:
-            export_xml.append(Expand(macro="stdio"))
+        if stdio_element:
+            try:
+                export_xml.append(stdio_element)
+            except Exception:
+                export_xml.append(Expand(macro="stdio"))
 
         # Append version command
         export_xml.append_version_command()
 
         if self.command_override:
-            command_line = self.command_override
+            command_text = self.command_override
         else:
-            command_line = []
+            command_text = []
             try:
-                command_line.append(export_xml.inputs.cli())
+                command_text.append(export_xml.inputs.cli())
             except Exception as e:
                 logger.warning(str(e))
                 raise
             try:
-                command_line.append(export_xml.outputs.cli())
+                command_text.append(export_xml.outputs.cli())
             except Exception:
                 pass
-        # Steal interpreter from kwargs
-        command_kwargs = {}
-        if export_xml.interpreter is not None:
-            command_kwargs["interpreter"] = export_xml.interpreter
-        # Add command section
-        command_node = etree.SubElement(export_xml.root, "command", **command_kwargs)
         if keep_old_command:
-            if getattr(self, "command", None):
-                command_node.text = etree.CDATA(export_xml.command)
+            if getattr(self, "command_line", None):
+                ctext = export_xml.command_text
```

Comment:
now that you've renamed command_line, this should be command_text, right?

```suggestion
            if getattr(self, "command_text", None):
                ctext = export_xml.command_text
```